yarn-project/noir-contracts.js/package.json

```diff
@@ -73,4 +73,4 @@
   "engines": {
     "node": ">=18"
   }
-}
\ No newline at end of file
+}
```

Comment:
🤷

Reply:
This a missing newline, the autoformatter was complaining about it